examples/cifar10-cnn/cnn_batchnorm.jl

```diff
@@ -0,0 +1,123 @@
+using Knet
+include(Pkg.dir("Knet", "data", "cifar.jl"))
+
+function loaddata()
+    info("Loading CIFAR 10...")
+    xtrn, ytrn, xtst, ytst, = cifar10()
+    mn = mean(xtrn, (1,2,4))
+    xtrn = xtrn .- mn
+    xtst = xtst .- mn
+    ytrn = onehot(Int.(ytrn))
```

Comment:
Do you need to use onehot?  We are using nll now, which takes integer answers.
Also if this mean subtraction thing is standard I can make it part of the cifar10() function.

Reply:
I used the old, familiar method to make things work quicker. `nll` will be more natural, so I'm planning to remove one-hot encoding.